app/Models/Rule.php

```
@@ -0,0 +1,125 @@
+    /**
+     * Get the execution logs for the rule.
+     */
+    public function executionLogs(): HasMany
+    {
+        return $this->hasMany(RuleExecutionLog::class);
+    }
+
+    /**
+     * Scope a query to only include active rules.
+     */
+    public function scopeActive($query)
```

PHPStan (2.1.15)

Method App\Models\Rule::scopeActive() has no return type specified.


(missingType.return)